bevy_proto_backend/src/proto/prototypes.rs

```diff
@@ -48,6 +48,26 @@ impl<'w, T: Prototypical> PrototypesMut<'w, T> {
         handle
     }
 
+    /// Load the prototypes at the given path.
+    ///
+    /// This will also store strong handles to the prototypes in order to keep them loaded.
+    ///
+    /// To load without automatically storing the handles, try using [`AssetServer::load_folder`].
+    pub fn load_folder<P: Into<Cow<'static, str>>>(&mut self, path: P) -> Result<Vec<Handle<T>>, AssetServerError> {
+        let path = path.into();
+        let handles = self.asset_server.load_folder(path.as_ref())?;
+        let handles: Vec<_> = handles.iter().map(|handle| handle.clone().typed::<T>()).collect();
+
+        handles.iter().for_each(|handle| {
+            // TODO: I'm not sure what to do here.
+            let path = self.asset_server.get_handle_path(handle).unwrap().path().to_str().unwrap();
```

Comment:
Is this even possible? I'm still learning Rust, but I'm not sure how `OsStr` could even be converted to `Cow<'static, str>`, but I'm probably missing something.